src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -154,59 +178,66 @@ private static Item getItem(@NotNull IBlockState blockState)
     {
         if (blockState.getBlock() instanceof BlockBanner)
         {
-            return Items.banner;
+            return Items.BANNER;
         }
         else if (blockState.getBlock() instanceof BlockBed)
         {
-            return Items.bed;
+            return Items.BED;
         }
         else if (blockState.getBlock() instanceof BlockBrewingStand)
         {
-            return Items.brewing_stand;
+            return Items.BREWING_STAND;
         }
         else if (blockState.getBlock() instanceof BlockCake)
         {
-            return Items.cake;
+            return Items.CAKE;
         }
         else if (blockState.getBlock() instanceof BlockCauldron)
         {
-            return Items.cauldron;
+            return Items.CAULDRON;
         }
         else if (blockState.getBlock() instanceof BlockCocoa)
         {
-            return Items.dye;
+            return Items.DYE;
         }
         else if (blockState.getBlock() instanceof BlockCrops)
         {
-            if (blockState.getBlock() instanceof BlockCarrot)
-            {
-                return Items.carrot;
-            }
-            else if (blockState.getBlock() instanceof BlockPotato)
+            ItemStack stack = ((BlockCrops) blockState.getBlock()).getItem(null, null, blockState);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'stack' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)